mesheryctl/pkg/utils/error.go

```diff
@@ -306,6 +304,52 @@ func ErrInvalidNameOrID(err error) error {
 		[]string{"Provided Name or ID would be invalid"}, []string{"Please Provide a valid name or ID by using list command with the appropriate subcommand"})
 }
 
+func ErrAttachAuthToken(err error) error {
+	return errors.New(ErrAttachAuthTokenCode, errors.Alert, []string{"Authentication token not found"},
+		[]string{"Authentication token not found", err.Error()}, []string{"User not logged in to generate token"}, []string{"Login with `mesheryctl system login` or supply a valid user token using the --token (or -t) flag."})
+}
+
+func ErrFailReqStatus(statusCode int) error {
+	return errors.New(ErrFailReqStatusCode, errors.Alert, []string{"Failed response Server Error"},
+		[]string{"Response Status Code " + strconv.Itoa(statusCode) + ", possible Server Error"}, []string{"Bad api call"}, []string{"Check your network connection and check the status of meshery server via 'mesheryctl system status'"})
+}
+
+func ErrProcessingConfig(err error) error {
+	return errors.New(ErrProcessingConfigCode, errors.Alert, []string{"Error processing config"},
+		[]string{"Error processing config", err.Error()}, []string{"Couldn't load mesh config"}, []string{"Ensure mesheryctl has the right configurations"})
+}
+
+func ErrMarshalIndent(err error) error {
+	return errors.New(ErrMarshalIndentCode, errors.Alert,
+		[]string{"Error indenting json body"}, []string{err.Error()}, []string{"Can't marshal indent body filters due to invalid content"}, []string{"Please check the data structure you are providing for Marshalling"})
```

Comment:
1. Don't say "can't". Use "Unable to".
2. What does "marshal indent body filters" mean? Rephrase this.
3. Don't say "Please". 
4. Don't say "You".
5. Don't capitalize "Marshalling".
6.

Reply:
Marshall Indent formats a value to a proper json format , but i will use a simpler and more concise word